playwright/src/main/java/com/microsoft/playwright/impl/junit/OptionsExtension.java

```diff
@@ -25,7 +25,8 @@
 import static com.microsoft.playwright.impl.junit.ExtensionUtils.*;
 
 public class OptionsExtension implements AfterAllCallback {
-  private static final ThreadLocal<Options> threadLocalOptions = new ThreadLocal<>();
+  // This is public to allow our tests to set the wsEndpoint dynamically
```

Comment:
Not sure, I understand why this needs to be public.

Reply:
The way the BrowserConnect tests are written, the wsEndpoint isn't known until after the Options are already created by the fixture.  Making this public is the only way i could think of to set the wsEndpoint in the BeforeAll of the test class.  I guess I could use reflection to do it.